src/settings.js

```diff
@@ -1635,6 +1635,10 @@ var FETCH = 0;
 // [link]
 var ASMFS = 0;
 
+// If set to 1, uses new filesystem implementation
```

Comment:
It would be good to add a note that this is still an experimental work in progress and is not expected to function correctly, probably with a link to the tracking issue.

Reply:
done